src/components/PageContent/ContentObserver/ContentObserver.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from "preact/hooks"
+import { useCurrentId } from "~/hooks/currentId/useCurrentId"
+import { updateTableOfContents } from "../../RightSidebar/TableOfContents/tocStore"
+import { useStore } from "@nanostores/preact"
+import { shouldUpdateCo } from "./coStore"
+
+export const ContentObserver = () => {
+  const { setCurrentId } = useCurrentId()
+  const $shouldUpdateCo = useStore(shouldUpdateCo)
```

Comment:
```suggestion
  const $shouldUpdateCo = useStore(shouldUpdateContentObserver)
```

Suggestion: make it more semantic. if this is only going to be used on the quickstart then it could be named `shouldUpdateQuickstartContentObserver`

Reply:
This is used in the docs currently, not the quickstarts. However I may replace the system I use in quickstarts with this